### PR TITLE
CSS: Views (Except Dashboard)

### PR DESCRIPTION
**Pull request by tdkent**
- updates responsive CSS for Home, Sign In, Request Service, and My Appointments views
- adds new splash image to Sign In and Home views
- delete "Cancel Form" view
- adds loading spinner component for suspense boundary fallback UI

### CODE REVIEW

**Review by jesslane94 — APPROVED** [review]
wow this looks amazing Tim O: I looked at in on mobile and changed the size of the screen and everything and it's so responsive. I like the grey box touch when the screen gets small behind the info on the landing page.

**Comment by tdkent** [comment]
> wow this looks amazing Tim O: I looked at in on mobile and changed the size of the screen and everything and it's so responsive. I like the grey box touch when the screen gets small behind the info on the landing page.

I forgot to mention in the description that I replaced the splash image only because the original is 800x800 and I needed something larger/wider for full screen desktop display.

**Comment by kristi-h** [comment]
All the pages are responsive to various device sizes and look good, as well as the splash image.
And I know you made this change previously, but the logo looks sooo good!

File: client/app/admin-dashboard/page.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default async function AdminDashboardView() {
   return (
     <div>
       <Typography variant="h1">Reservations:</Typography>
-      <Suspense fallback={<CircularProgress />}>
+      <Suspense fallback={<Spinner />}>
```
Comment (kristi-h) [inline client/app/admin-dashboard/page.jsx:25]: Why is using Spinner better than using Circular Progress?
Does it just look better?

**Review by kristi-h — APPROVED** [review]

File: client/app/admin-dashboard/page.jsx (hunk 1 above)
Comment (tdkent) [reply client/app/admin-dashboard/page.jsx:25]: The Spinner component uses the CircularProgress component within it, it just puts into a Box  for better formatting on the page.

**Pull request merged**